kotlin-sdk/app/src/main/java/com/mongodb/app/ItemActivity.kt

```diff
@@ -115,31 +115,40 @@ class TodoActivity : AppCompatActivity() {
     }
 
     /**
-     *  Creates a popup that allows the user to insert a todo into the realm.
+     *  Creates a popup that allows the user to insert a item into the realm.
      */
     private fun onFabClicked() {
-        val input = EditText(this)
-        val dialogBuilder = AlertDialog.Builder(this)
-        dialogBuilder.setMessage("Enter todo name:")
-                .setCancelable(true)
-                .setPositiveButton("Create") { dialog, _ ->
-                    run {
-                        dialog.dismiss()
-                        val todo = Todo(realmApp.currentUser!!.identity)
-                        todo.summary = input.text.toString()
-                        CoroutineScope(Dispatchers.IO).launch {
-                            realm.write {
-                                this.copyToRealm(todo)
-                            }
+        val mBuilder = AlertDialog.Builder(this)
+            .setMessage("Enter Item Name:")
+        val mView: View = layoutInflater.inflate(R.layout.create_item_dialog, null)
+
+        val itemSummaryInput = mView.findViewById<View>(R.id.plain_text_input) as EditText
+
+        mBuilder
+            .setPositiveButton("Create") { dialog, _ ->
+                run {
+                    val item = Item(realmApp.currentUser!!.identity)
+                    item.summary = itemSummaryInput.text.toString()
+
+                    CoroutineScope(Dispatchers.IO).launch {
+                        realm.write {
+                            this.copyToRealm(item)
                         }
                     }
+
+                    // Display the item created using Android's Toast feedback popup
+                    Toast.makeText(
+                        this,
+                        "Item created: ${item.summary}",
+                        Toast.LENGTH_SHORT
+                    ).show()
                 }
             .setNegativeButton("Cancel") { dialog, _ ->
                     dialog.cancel()
                 }
         val dialog = dialogBuilder.create()
         dialog.setView(input)
-        dialog.setTitle("Add ToDo")
+        dialog.setTitle("Add Item")
```

Comment:
FWIW in the RN app I have this dialog titled as "Add To-Do Item" + left the button as "Add To-Do" + titled the screen "Your To-Do Items"